e2e-tests/tests/users/09-read-from-primary-custom-db.yaml

```diff
@@ -7,7 +7,7 @@ commands:
       set -o xtrace
 
       source ../../functions
-
-      data=$(run_psql 'SELECT * from customApp;' "rhino:'$(get_psql_user_pass rhino-credentials)'@$(get_psql_user_host rhino-credentials):5432/zoo")
+      password=$(get_psql_user_pass rhino-credentials)
+      data=$(run_psql 'SELECT * from customApp;' "rhino:$password@$(get_psql_user_host rhino-credentials):5432/zoo")
```

Comment:
Why do not we use `pgbouncer-uri` from the secret to connect to DB? We connect to the primary without proxy.  We need to use proxy in tests.

Reply:
I will check it. I thought it was done by some reason